src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -56,7 +56,8 @@ def load_command_table(self, _):
         g.custom_command('start', 'app_start', supports_no_wait=True)
         g.custom_command('stop', 'app_stop', supports_no_wait=True)
         g.custom_command('restart', 'app_restart', supports_no_wait=True)
-        g.custom_command('log tail', 'app_tail_log')
+        g.custom_command('log', 'app_tail_log')
+        g.custom_command('log tail', 'app_tail_log', deprecate_info='az spring-cloud app log')

```

Comment:
Add more info to deprecate_info such as redirect, hide, expiration. You can refer to docs in [https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments)

Reply:
Have done, thanks